pack/apk.mk

```diff
@@ -45,12 +45,12 @@ package: $(BUILDDIR)/$(TARBALL) \
 	@echo "-------------------------------------------------------------------"
 	sudo apk update
 	mkdir ~/.abuild/
-	echo "PACKAGER_PRIVKEY=/home/${USER}/.abuild/private.rsa" > ~/.abuild/abuild.conf
+	echo "PACKAGER_PRIVKEY=~/.abuild/private.rsa" > ~/.abuild/abuild.conf
```

Comment:
Usually `~` is expanded in a shell and programs don't expect it in paths. If it works, okay. However I would use `$HOME` instead.

I tried to verify it with tarantool with https://github.com/tarantool/tarantool/pull/7942 (applied on top of current master), but got:

```
/build/src/tarantool-2.11.0_alpha0_p805/third_party/luajit/src/lj_err.c: In function 'lj_err_unwind_dwarf':
/build/src/tarantool-2.11.0_alpha0_p805/third_party/luajit/src/lj_err.c:413:2: error: #error "Broken build system -- only use the provided Makefiles!"
  413 | #error "Broken build system -- only use the provided Makefiles!"
      |  ^~~~~
```

It fails before signing the package, so I can't tell, whether the change works.

Reply:
I replaced "~" to $HOME variable. But I don't think that this error connected with packpack.